build.gradle

```diff
@@ -187,34 +189,40 @@ ideaModule.dependsOn swaggerSources.server.code
 // This is the path to the default Google service account for the janitor service to run as.
 def googleCredentialsFile = "${projectDir}/src/test/resources/rendered/sa-account.json"
 
-test {
-    //  Sets the application Google Credentials.
+tasks.withType(Test) {
     environment = [
             'GOOGLE_APPLICATION_CREDENTIALS': "${googleCredentialsFile}"
     ]
-    useJUnitPlatform()
+    testLogging {
+        events = ["passed", "failed", "skipped"]
+        // Causes the correct line to be reported on an exception.
+        exceptionFormat = TestExceptionFormat.FULL
+        reports {
+            // Write XML file (used by CircleCI, Jenkins, etc) to api/build/test-results/test
+            junitXml.enabled = true
+            // Write human-readable test report to api/build/reports/
```

Comment:
what is the api direcotry? I don't think we have the directory yet. How about customize our own report direcroty under build dir? 
I think this may work:
```
 reports.junitXml.destination = file("$buildDir/test-results/test")
 reports.html.destination = file("$buildDir/reports")
```

Reply:
Oops – that was copy-pasta from the AoU repo where the base directory was 'api'. The default directories make sense, and I updated the comment to accurately reflect where they go.